### ISSUE TITLE
Interfacing the GSM Module

### ISSUE THREAD

**Issue by mikeshams** [lokeshchinthakuntla/Project#5]
Lokesh - 1) What issues did you face when Interfacing the GSM Module to the LPC1768? 2) how did you resolve them?

**Comment by lokeshchinthakuntla** [comment]
while interfacing the Gsm module with LPC1768. we came across a problem that we are unable to get the text message from the gsm module. when we run the code there are no errors and we got the binary file successfully. we can see the AT commands executing perfectly in the serial monitoring system. But, when the garbage level reached above the threshold value SMS from the GSM module is not generated. To solve this problem we used Arduino in between LPC1768 and GSM module. when the garbage level reaches 75%, For sending AT commands and communicating with the SIM800L module, we will use the serial monitor. With the help of the code, commands will enable the Arduino to communicate with the SIM800L module on the serial monitor. we give a high signal to Arduino. In functions, we initialize a serial communication link between Arduino, Arduino IDE, and SIM800L module at a baud rate of 9600.gsm sends text messages to the user.

**Comment by lokeshchinthakuntla** [comment]
while interfacing the Gsm module with LPC1768. we came across a problem that we are unable to get the text message from the gsm module. when we run the code there are no errors and we got the binary file successfully. we can see the AT commands executing perfectly in the serial monitoring system. But, when the garbage level reached above the threshold value SMS from the GSM module is not generated. To solve this problem we used Arduino in between LPC1768 and GSM module. when the garbage level reaches 75%, For sending AT commands and communicating with the SIM800L module, we will use the serial monitor. With the help of the code, commands will enable the Arduino to communicate with the SIM800L module on the serial monitor. we give a high signal to Arduino. In functions, we initialize a serial communication link between Arduino, Arduino IDE, and SIM800L module at a baud rate of 9600.gsm sends text messages to the user

**Comment by lokeshchinthakuntla** [comment]
On further analysis, we got another code for gsm with LPC while compiling this code we got the message from gsm which is perfect so implemented this code in our main project file and now it works perfectly.this code we got it from internet we first defined the number of the gsm module and changed the message according to our use.we did not found any issues it runs perfectly for the first without any errors in the code while compiling.  with this code, there's no need of using Arduino in between. by decreasing the number of components we can avoid all the complexity and also budget-friendly.
